utils/kafka-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaConfig.scala

```diff
@@ -49,6 +60,12 @@ case class KafkaConfig(
 
 }
 
+private object KafkaEspPropertiesConfig {
```

Comment:
Please also change the name to KafkaEspPropertiesConfig? We don't need here Esp ;)

Reply:
The namespace should be written in a constant style: https://docs.scala-lang.org/style/naming-conventions.html#constants-values-and-variables. I wonder, it should also be public / available to external projects?

Reply:
I didn't want to name it KafkaPropertiesConfig not to potentially confuse someone that this is set in that map (kafkaProperies is not kafkaEspProperies). WDYT about just moving it to KafkaConfig as public constants?